pkg/controllers/nodeclaim/disruption/drift.go

```diff
@@ -71,12 +71,8 @@ func (d *Drift) Reconcile(ctx context.Context, nodePool *v1.NodePool, nodeClaim
 	nodeClaim.StatusConditions().SetTrueWithReason(v1.ConditionTypeDrifted, string(driftedReason), string(driftedReason))
 	if !hasDriftedCondition {
 		log.FromContext(ctx).V(1).WithValues("reason", string(driftedReason)).Info("marking drifted")
-		metrics.NodeClaimsDisruptedCounter.With(prometheus.Labels{
-			metrics.TypeLabel:     metrics.DriftReason,
-			metrics.NodePoolLabel: nodeClaim.Labels[v1.NodePoolLabelKey],
-		}).Inc()
-		metrics.NodeClaimsDriftedCounter.With(prometheus.Labels{
-			metrics.TypeLabel:     string(driftedReason),
+		metrics.NodeClaimsDisruptedTotal.With(prometheus.Labels{
```

Comment:
This should be dropped, right? Since we are now tracking this metric through `operator_status_condition_transitions_total`?